docs/src/modules/components/MarkdownDocs.js

```diff
@@ -38,10 +41,39 @@ const styles = theme => ({
     marginBottom: theme.spacing(2),
     padding: theme.spacing(0, 1),
   },
+  footer: {
+    marginTop: theme.spacing(16),
```

Comment:
Too much whitespace, although I'll concede that `theme.spacing(4)`, was perhaps too little. How does 8 `look`?

Reply:
I think it looks better with `20`. I have found a compromise.

Reply:
I'll trade you grey buttons and white background for less whitespace. 😄 

I had `4`, you like `20`, so half the difference is `12`.

That actually creates a more balanced appearance with the space above the H1.

Reply:
👍